src/index.ts

```diff
@@ -45,6 +46,7 @@ export const COMMANDS = {
   'openapi:inspect': OpenAPIInspectCommand,
   'openapi:reduce': OpenAPIReduceCommand,
   'openapi:validate': OpenAPIValidateCommand,
+  'openapi:refs': OpenAPISolvingCircularityAndRecursiveness,
```

Comment:
alphabetizing and renaming:

```suggestion
  'openapi:refs': OpenAPIRefsCommand,
  'openapi:validate': OpenAPIValidateCommand,
```